app.py

```diff
@@ -113,7 +101,7 @@ def start():
 
 @app.route("/leaderboard") #leaderboard
 def hello_world():
-    return render_template("leaderboard.html", data = data)
+    return render_template("leaderboard.html", data=data)
```

Comment:
```suggestion
    return render_template("leaderboard.html", data = func())
```
@ProPython007 check if this thing makes our updates close to realtime => everytime someone visits `/leaderboard` the code to fetch fresh data would re-run.

Reply:
@akshitadixit It takes too much time loading that page..... even a simple refresh too...... As heroku re-run everytime for new requests so ig data=data is ok here.....